pkg/network/openshift_sdn_test.go

```diff
@@ -111,7 +112,7 @@ func TestFillOpenShiftSDNDefaults(t *testing.T) {
 	// vars
 	f := false
 	p := uint32(4789)
-	m := uint32(1450)
+	m := uint32(8050)
```

Comment:
Is this a typo?

Reply:
nope; just testing that a theoretical mtu of 9000 is handled correctly.

Reply:
er, yes, it was a typo. should be 8950